scripts/data_files/driver_jsons/driver_opaque_schema.json

```
@@ -0,0 +1,78 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "_comment": {
+      "type": "string"
+    },
+    "prefix": {
+      "type": "string",
+      "pattern": "^[A-Z_a-z][0-9A-Z_a-z]*$"
+    },
+    "type": {
+      "type": "string",
+      "const": ["opaque"]
+    },
+    "location": {
+      "type": ["integer","string"],
+      "pattern": "^(0x|0X)?[a-fA-F0-9]+$"
+    },
+    "mbedtls/h_depend_on": {
+      "type": "string"
+    },
+    "headers": {
+      "type": "array",
+      "items": {
+          "type": "string"
+        },
+        "minItems": 1,
+        "uniqueItems": true
+    },
+    "capabilities": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "_comment": {
+              "type": "string"
+            },
+            "mbedtls/c_depend_on": {
+              "type": "string"
+            },
+            "entry_points": {
+              "type": "array",
+              "items": {
+                  "type": "string",
+                  "enum": ["import_key", "export_key", "export_public_key",
+                          "copy_key", "get_builtin_key"]
+                },
+                "minItems": 1,
+                "uniqueItems": true
+            },
+            "name": {
```

```suggestion
            "names": {
```

Goes for both types of drivers.

There's one place in `psa-driver-interface.md` with this typo as well, please fix it.